src/app/todo/todo.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { TodoServiceService } from '../services/todo-service.service';
+
+@Component({
+  selector: 'app-todo',
+  templateUrl: './todo.component.html',
+  styleUrls: ['./todo.component.css']
+})
+export class TodoComponent implements OnInit {
+
+  todoList: any= [];
+  storeList = [];
+  errMsg: string = "No task";
+  dataObj: any= {};
+
+  constructor( private taskData:TodoServiceService) { }
```

Comment:
It's better to name the variable of service name almost similar to the Service. This keeps clarity.

Reply:
okay